docs/reference/migration/migrate_8_0/cluster-node-setting-changes.asciidoc

```diff
@@ -907,3 +907,29 @@ will continue to use that configured value.
 Any node that did not have an explicitly configured password hashing algorithm in
 {es} 6.x or {es} 7.x would have failed to start.
 ====
+
+//tag::notable-breaking-changes[]
+.The `xpack.monitoring.history.duration` will not delete indices created by metricbeat or elastic agent
+[%collapsible]
+====
+*Details* +
+
+Prior to 8.0, Elasticsearch would internally handle removal of all monitoring indices according to the
+`xpack.monitoring.history.duration` setting.
+
+When using metricbeat or elastic agent >= 8.0 to collect monitoring data, indices are managed via an ILM policy. The policy will be created using the `xpack.monitoring.history.duration` as an initial retention period.
```

Comment:
Maybe a small change to highlight that it picks up the property if it's set, but doesn't need it?
```suggestion
When using metricbeat or elastic agent >= 8.0 to collect monitoring data, indices are managed via an ILM policy. If the setting is present, the policy will be created using the `xpack.monitoring.history.duration` as an initial retention period.
```